<ui version="4.0" >
 <class>PrecisionCheckerClass</class>
 <widget class="QMainWindow" name="PrecisionCheckerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PrecisionChecker</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QFrame" name="frameImagePanel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>901</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadImage" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text" >
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPage" >
    <property name="geometry" >
     <rect>
      <x>990</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <item>
     <property name="text" >
      <string>Page 0</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="chkViewBit" >
    <property name="geometry" >
     <rect>
      <x>990</x>
      <y>110</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>ViewThreshHold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkFlexArea" >
    <property name="geometry" >
     <rect>
      <x>990</x>
      <y>140</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>ViewFlexArea</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Param</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>620</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>grp1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Angle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Focus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHistgram" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Histgram</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>560</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QSpinBox" name="txtX_2" >
       <property name="maximum" >
        <number>20000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="txtY_2" >
       <property name="maximum" >
        <number>20000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>530</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QSpinBox" name="txtX_1" >
       <property name="maximum" >
        <number>20000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="txtY_1" >
       <property name="maximum" >
        <number>20000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_7" >
    <property name="geometry" >
     <rect>
      <x>980</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>ReScan</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
